### DISCUSSION TITLE
jQuery and Bootstrap old version

### DISCUSSION THREAD

**Discussion by Sysadminfromhell** [modoboa/modoboa#2170] [General]
According to the Website www.immuniweb.com/websec/ the Modoboa Webmail Website uses a old jQuery and a old Bootstrap Version:

`jQuery 1.9.1
The fingerprinted component version is outdated and vulnerable to publicly known vulnerabilities. Urgently update to the most recent version 3.5.1. `

`Bootstrap 3.3.7
The fingerprinted component version is outdated and vulnerable to publicly known vulnerabilities. Urgently update to the most recent version 4.6.0. `

Is there any way to Update it?

**Comment by tonioo** [comment]
Indeed, those versions are outdated but a complete interface rewrite has started and it doesn't use jquery or bootstrap anymore. I don't know when it'll be ready but I don't want to spend time upgrading bootstrap.

**Reply by Sysadminfromhell** [reply]
Sure thing, if there a possibility to help with Test so tell me. I'd like to help.
